tests/shared/src/test/scala/cats/effect/std/QueueSpec.scala

```diff
@@ -124,7 +124,7 @@ class BoundedQueueSpec extends BaseSpec with QueueTests[Queue] {
     }
 
     "offer/take at high contention" in real {
-      val size = 100000
+      val size = 10000
```

Comment:
If you want to keep it big for JVM.

```suggestion
val size = if (cats.platform.Platform.isJVM) 100000 else 10000
```

Reply:
Wait `cats.platform.Platform` is public?

Reply:
It comes from `cats-kernel-laws` and is `private[cats]`. Good for us.